### DISCUSSION TITLE
VIP subnet issue (set as /32 when it should be /24)

### DISCUSSION THREAD

**Discussion by steubens** [siderolabs/talos#9216] [Q&A] [answered]
Hello - I am deploying a cluster, and for whatever reason the VIP is showing up on the Talos Controlplane VM as 10.8.8.8/32 when the node's primary network interface is a /24. Therefore, the VIP is unable to be reached by my laptop (or any other machines) to continue the bootstrapping process - the VIP is on the wrong subnet (/32)... I don't see a way to set the VIP subnet manually...

Any ideas?? Thank you in advance! 

---
Relevant part of my host-config.yaml:

```sh
network:
    hostname: talos8811
    interfaces:
      - deviceSelector:
          hardwareAddr: bc:24:11:af:f6:f4
        addresses:
          - 10.8.8.11/24
        routes:
          - network: 0.0.0.0/0
            gateway: 10.8.8.1
        mtu: 1500
        dhcp: false
        vip:
          ip: 10.8.8.8
```

Node holding the VIP as a /32 and unable to be reached on that IP:
![CleanShot 2024-08-22 at 17 16 31@2x](https://github.com/user-attachments/assets/97389300-a5fe-4b01-8034-fa4ed2ba5f73)

**Comment by steubens** [comment, marked as answer]
I finally figured this out on my own... and it took so long because I'm a networking dummy.  I ended up adding a route just for the 10.8.8.8/32 subnet in my unifi router and that did the trick!

**Reply by smira** [reply]
VIP is always /32, as it adds to your base network address, it doesn't replace it.
